src/tracks/graph/interfaces.ts

```diff
@@ -5,23 +5,63 @@ import { Plot } from '../../plots';
 
 export type PlotCreatorFunction = (config: PlotConfig, trackScale: D3Scale) => Plot;
 
+/**
+ * Config object used to define a plot in the graph track options
+ */
 export interface PlotConfig {
   id: string|number,
   type: string,
   options: PlotOptions,
 }
 
+/**
+ * Factory object for creating an instance of a plot given a type
+ */
 export interface PlotFactory {
   [propName: string]: PlotCreatorFunction,
 }
 
+/**
+ * Available graph track options
+ */
 export interface GraphTrackOptions extends TrackOptions {
+  /**
+   * Collection of plot config objects
+   */
   plots?: PlotConfig[],
+  /**
+   * Data for all plots in the track. May be of any type or a function or promise
+   * returning data. The plots will need to have a data accessor function defined, that
+   * can pick the data it needs from this value.
+   */
   data?: Promise<any> | Function | any,
+  /**
+   * Whether or not to show the loader or not (if configured)
+   */
   showLoader?: boolean,
+  /**
+   * Scale type to use. This scale applies to all plots unless the plot config has its
+   * own scale and domain set. Can be 'linear' or 'log'.
+   */
   scale?: string,
+  /**
+   * Domain to use for the graph data range
+   */
   domain?: Domain,
+  /**
+   * Determines if the legend should be clickable to toggle plots on/off when used within a
+   * TrackGroup with graphLegendConfig enabled.
```

Comment:
i assume default is false
```suggestion
   * TrackGroup with graphLegendConfig enabled. Default: false
```

Reply:
Added default, which is true